### PR TITLE
Paper patches

### PR DESCRIPTION
**Pull request by ct-clmsn**
@SAtacker feel free to comment on this PR before I do the merge!

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -19,6 +19,7 @@ set(CHPLX_VERSION
 )
 
 set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_FLAGS_INIT "-std=c++20")
```
Comment (SAtacker) [inline CMakeLists.txt:22]: Why do we need this?

File: frontend/lib/util/git-version.cpp [hunk 2]
```diff
@@ -1,4 +1,3 @@
 namespace chpl {
-  const char* GIT_SHA = "xxxxxxxxxx";
+  const char* GIT_SHA = "ab869c86";
```
Comment (SAtacker) [inline frontend/lib/util/git-version.cpp:2]: Can we please not change this file?

File: publications/fall2023/fall2023paper.def [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+./apex_exec --apex:otf2 --apex:csv --apex:taskgraph --apex:scatter --apex:postprocess --apex:source --apex:cpuinfo --apex:meminfo --apex:status fibonacci
+cd ../..
+
+git clone https://github.com/fmtlib/fmt.git
+mkdir fmt/build
+cd fmt/build
+cmake -DCMAKE_CXX_COMPILER=clang++-15 -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH=$PWD/../../opt .. && make -j && make install
+cd ../..
+
+export PATH=$PWD/opt/bin:$PATH
```
Comment (SAtacker) [inline publications/fall2023/fall2023paper.def:167]: It should be `/opt/chapel/bin` I guess?

Comment (SAtacker) [inline publications/fall2023/fall2023paper.def:167]: I think it should be `/opt/chapel/bin` path

**Review by SAtacker — CHANGES_REQUESTED** [review]
Also we need to justify the different cxx versions wherever required just as a comment to make a note of it. Thanks!

File: CMakeLists.txt (hunk 1 above)
Comment (ct-clmsn) [reply CMakeLists.txt:22]: without that flag cmake slips in a compile time flag (similar to experimental) that breaks the build.

**Comment by ct-clmsn** [comment]
@SAtacker I'll update the test validation system this evening and merge the PR when that's done.

**Comment by ct-clmsn** [comment]
@SAtacker merge completed.

**Comment by SAtacker** [comment]
> @SAtacker merge completed.

thank you!

**Pull request merged**